.github/workflows/tests.yml

```diff
@@ -167,11 +167,15 @@ jobs:
     - name: Install dependencies
       run: |
         composer validate
-        composer install --prefer-dist --no-progress --no-suggest
```

Comment:
@mabumusa1 is there a reason `--prefer-dist` is not possible anymore? as I see no composer changes that require this

Reply:
@mollux good catch, I changed this line while testing, and reverted back on commit c33aa03